spec/features/home_page_spec.rb

```diff
@@ -1,6 +1,45 @@
-feature 'Home Page visit', js: true do
-  scenario 'Unregistered visitor visits home page' do
-    visit '/'
-    expect(page).to have_content 'Hello World!'
+feature 'Home Page interaction', js: true do
+
+  before :each do
+    @home_page = HomePage.new
+    @home_page.load
+  end
+
+  scenario 'Unregistered visitor got redirected to welcome page' do
+    @welcome_page = WelcomePage.new
+    expect(@welcome_page).to be_displayed
+  end
+
+  scenario 'Unregistered visitor is capable of signing up' do
+    user = FactoryBot.build(:user)
+
+    @home_page.menu.sign_up_link.click
+
+    @sign_up_page = SignUpPage.new
+    expect(@sign_up_page).to be_displayed
+
+    @sign_up_page.sign_up(user)
+    expect(page).to have_content 'A message with a confirmation link has been sent to your email address. Please follow the link to activate your account'
```

Comment:
This line overlaps the limit of symbols too much. Probably, something went wrong with a rubocop metrics. Check it, please.